scripts/funcTests/runFuncTests.sh

```diff
@@ -37,22 +37,28 @@ curl -o cli/dotnet-install.sh https://dot.net/v1/dotnet-install.sh
 # Run install.sh
 chmod +x cli/dotnet-install.sh
 
-# v1 needed for some test and bootstrapping testing version
-cli/dotnet-install.sh -i cli -c 1.0
+# Get recommended version for bootstrapping testing version
+cli/dotnet-install.sh -i cli -NoPath
```

Comment:
@zivkan made a change here I think.

Reply:
Yes, I took @zivkan 's change and appended -NoPath on this change. 
If I don't append -nopath, the dotnet msbuild will build the projects using the latest version under cli, but not the stable version installed in yaml task.